### ISSUE TITLE
discrepancy between relative abundance and scale in heat map?

### ISSUE THREAD

**Issue by rebeccaco** [microbiome/microbiome#86]
Hello, 

I am trying to generate a heat map for the core microbiota of my amplicon sequencing data. I was following the tutorial on Github and on Bioconductor and noticed that the x-axis scales in your tutorials might be 2 magnitudes lower than what the x-axis has been labelled.

For example: 
In this [tutorial](https://microbiome.github.io/microbiome/Core.html), the code for generating a heat map for the core microbiota is as follows:

# generate relative abundance*
pseq.rel <- microbiome::transform(pseq, "compositional")

# compositions for heat map
prevalences <- seq(.05, 1, .05)
detections <- 10^seq(log10(1e-5), log10(.2), length = 10)

# visualize heat map
gray <- gray(seq(0,1,length=5))
p <- plot_core(pseq.rel, plot.type = "heatmap", colours = gray,
    prevalences = prevalences, detections = detections, min.prevalence = .5) +
    xlab("Detection Threshold (Relative Abundance (%))")
print(p)   

*the thing I don't understand here is that the relative abundance generated by "transform" in pseq.rel are not in percentages as the manual indicates. It's only in relative abundance which means it has to be multiplied by 100 for it to be considered "relative abundance (%)" as labelled on the x-axis of those heat maps. The scale would also need to be adjusted accordingly. 

 I was wondering if this is an issue I am misunderstanding or is this an error in the code? Thank you for your help.

Rebecca

**Comment by antagomir** [comment]
Thanks for pointing this out. I have now clarified the documentation and changed the scale in plot_core heatmap output into actual percentages in [0, 100]%. These changes are now in the Github devel branch. Will later become merged in master and online tutorials during the next update rounds.

**Comment by rebeccaco** [comment]
Hi Dr. Lahti,

Thank you for the fix. I am trying to incorporate your fix so that I can plot my data now. I was wondering how to go about changing the x-axis scale so that it goes up to relative abundance of 100% as seen in the Shetty _et al_. 2017 paper? My scale goes up to 10% and it does not show the prevalence petering off for the most abundant organisms at it reaches 100%. I tried fiddling with the upper detection limit but had no success. 

Thank you very much for your help and time.

**Comment by antagomir** [comment]
Hmm did you try to use ggplot options such as p + scale_x_log10(labels = scales::percent, limits = c(...))

for instance
```

library(microbiome)
data(atlas1006)
pseq <- subset_samples(atlas1006, DNA_extraction_method == 'r')
p <- plot_core(transform(pseq, "compositional"), plot.type = "heatmap") +
       scale_x_log10(labels = scales::percent, limits = c(1e-3, .5))
print(p)

```

**Comment by rebeccaco** [comment]
Hi Dr. Lahti,

I tried your example but the scale also didn't go up to 100% which is the same problem I am having. Below is the plot I got for the dataset atlas1006. I am unsure of how to change the end limit in scale_x_log10. If I put in .8, it reaches 100% but the scale now only has ticks at 1% and 100%. 

![image](https://user-images.githubusercontent.com/42045984/43645195-31312886-96ff-11e8-9370-369ea7e528fd.png)
